client/src/app/site/assignments/components/assignment-detail/assignment-detail.component.ts

```diff
@@ -111,6 +112,16 @@ export class AssignmentDetailComponent extends BaseComponent implements OnInit {
      */
     private _assignment: ViewAssignment;
 
+    public get candidates(): ViewUser[] {
+        /**
+         * FIXME:
+         * Confirm: Not sure if candidatesAsUsers or candidates (as AssignmentCandidate)
```

Comment:
`candidatesAsUsers` is ok: It is just a `map` from the m2m model (`ViewAssignmentCandidate`) to the linked `ViewUser`. It's should be correctly sorted[1]. So if you just want to display, it is enough. For sorting the m2m model should be used.

[1] I just checked the `relations.ts`: The order is missing. I'll add it to this PR.

Reply:
Done.